gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -526,9 +528,11 @@ MM_VerboseHandlerOutputStandard::handleConcurrentMarkEnd(J9HookInterface** hook,
 	MM_ConcurrentGCStats *collectionStats = stats->_collectionStats;
 	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
 	MM_VerboseWriterChain* writer = _manager->getWriterChain();
+	MM_GCExtensionsBase *extensions = MM_GCExtensionsBase::getExtensions(env->getOmrVM());
+	MM_Scavenger *scavenger = extensions->scavenger;
 
 	uint64_t duration = 0;
-	bool deltaTimeSuccess = getTimeDeltaInMicroSeconds(&duration, stats->_startTime, stats->_endTime);
+	bool deltaTimeSuccess = getTimeDeltaInMicroSeconds(&duration, scavenger->_cycleTimes.cycleStartTime, scavenger->_cycleTimes.cycleEndTime);
```

Comment:
This doesn't seem correct, this method is Concurrent Mark specific, is there a reason to fetch scavenger times here?